engine/src/test/java/com/stratio/decision/functions/FunctionsTestIT.java

```diff
@@ -115,7 +116,7 @@ public void testSaveToCassandra() throws Exception {
         LOGGER.debug("Connecting to Cassandra Quorum: " + conf.getStringList("cassandra.hosts").toString());
 
         SaveToCassandraActionExecutionFunction func = new SaveToCassandraActionExecutionFunction(
-                getHostsStringFromList(conf.getStringList("cassandra.hosts")), ProtocolOptions.DEFAULT_PORT);
+                getHostsStringFromList(conf.getStringList("cassandra.hosts")), ProtocolOptions.DEFAULT_PORT, 50,  BatchStatement.Type.UNLOGGED);
```

Comment:
I would add defaultBatch size as a constant and detail the default value in conf file